index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="my-project">
+      <h1>Projects</h1>
+      <img src="images/Rectangle 21.png" alt=""  class="project1"/>
+      <div class="works">
```

Comment:
Hello @elarhadu well done with your project! Please keep in mind that you should _add_ the projects/about sections to your previous sections 😉. Kindly add all your sections together.